dev/App.vue

```diff
@@ -91,6 +91,8 @@
           :date-one="inlineDateOne"
           :months-to-show="2"
           :disabled-dates="['2018-04-30', '2018-05-10', '2018-12-14']"
+          :customized-dates="[['2019-03-21', '2019-03-22', '2019-03-23'], ['2019-03-21', '2019-03-22', '2019-03-23']]"
+          :customized-groups="['booked', 'not-available']"
```

Comment:
@MikaelEdebro not sure if this is the best approach (using two arrays for one purpose)... Hash maybe?
`customized-dates="['booked': ['2019-03-21', '2019-03-22', '2019-03-23'], 'not-available': ['2019-03-21', '2019-03-22', '2019-03-23']]`